src/main.ts

```diff
@@ -1,31 +1,30 @@
 import * as core from '@actions/core'
 import {Inputs, createPullRequest} from './create-pull-request'
 import {inspect} from 'util'
-import * as utils from './utils'
 
 async function run(): Promise<void> {
   try {
     const inputs: Inputs = {
       token: core.getInput('token'),
       path: core.getInput('path'),
-      addPaths: utils.getInputAsArray('add-paths'),
+      addPaths: core.getMultilineInput('add-paths'),
```

Comment:
`getMultilineInput` is not a replacement for my implementation. It does not parse comma-separated values. This action used comma-separated values in a much older version and I've kept that functionality for backwards compatibility. I cannot remove this without causing a breaking change. At some point in the future I may consider it when bumping the major version, but for now please revert this back.

Reply:
Ah, I didn't notice that. Will revert this.

Reply:
Reverted now.